### ISSUE TITLE
Low test coverage

### ISSUE THREAD

**Issue by estambakio-sc** [OpusCapita/react-crudeditor#98]
Due to a huge refactoring we have a very low test coverage: branches ~14%, others ~30-40%. 

We need to write more tests and bump all numbers up to 80% or higher.

**Comment by estambakio-sc** [comment]
After #198 the coverage is the following: 

Statements   : 75.06% ( 2477/3300 )
Branches     : 42.68% ( 452/1059 )
Functions    : 69.18% ( 687/993 )
Lines        : 76.3% ( 2412/3161 )

There's still work to be done.

**Comment by estambakio-sc** [comment]
I'm gonna close it because future improvements in test coverage should be a natural part of other issues, and keeping it as a separate issue makes little sense.
